train.ipynb

```diff
@@ -0,0 +1,248 @@
+        "*Installing git LFS*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "L0P6khtrFXpW",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!cd git-lfs-2.4.2/ && ./install.sh"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "kgzSKuXUFZi0",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!git lfs install"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "nJ31i4Z8JHa8",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Cloning the repository*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "NPOQ5rFHFbnU",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!git lfs clone --recurse-submodules https://github.com/marco-c/autowebcompat.git"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "8E5K5gxGJKrW",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Marking working directory*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "GIxBvGGdFdoK",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "os.chdir('autowebcompat')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ugZB2EhbJOui",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Installing requirements*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "wBqw99O-FfTK",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!pip install -r requirements.txt"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "SREO2hv9FhgE",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!pip install -r test-requirements.txt"
```

Comment:
We shouldn't need to install test-requirements.txt, but we can remove this in a follow-up. I'll file an issue.